aries-site/src/data/structure.js

```diff
@@ -323,6 +340,11 @@ export const structure = [
     seoDescription:
       'Used with text based navigation, such as inline text, header navigation, and footer navigation.',
     sections: [],
+    previewComponent: {
+      component() {
```

Comment:
any reason this one doesn't follow same `() => <AmchorPreview />` structure as others?

Reply:
nope lol just had that at the beginning and forgot to change back